raiden/tests/utils/transfer.py

```diff
@@ -780,6 +801,24 @@ def assert_synced_channel_state(
     )
 
 
+def assert_transfer_happy_path_invariants(
```

Comment:
This should either be renamed to *not* have `path` in the name, or the assert should be done on the path.